01-CORE-SYSTEMS/01-ORCHESTRATION/DEEP_AGENTS/system_prompts/__init__.py

```
@@ -18,7 +18,7 @@
 from .prompt_validator import PromptValidator
 
 __all__ = [
-    "TrustAwarePromptManager",
+    "your_secret_here",
     "PromptTemplates",
     "PromptValidator"
 ]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)